<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>331</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QHoverSensitiveButton" name="rightButton">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>120</y>
       <width>112</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>RIGHT</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>251</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QHoverSensitiveButton" name="leftButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>112</width>
       <height>61</height>
      </rect>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>LEFT</string>
     </property>
    </widget>
    <widget class="QHoverSensitiveButton" name="downButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>170</y>
       <width>112</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>DOWN</string>
     </property>
    </widget>
    <widget class="QHoverSensitiveButton" name="upButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>71</y>
       <width>112</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>UP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QHoverSensitiveButton" name="beginButton">
     <property name="geometry">
      <rect>
       <x>41</x>
       <y>70</y>
       <width>321</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Begin</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QHoverSensitiveButton" name="hoverButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>180</y>
       <width>201</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string>Hover Mode?</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QHoverSensitiveButton</class>
   <extends>QPushButton</extends>
   <header>qhoversensitivebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
